contrib/fwupd.spec.in

```diff
@@ -249,12 +247,8 @@ fwupd wrapper for Qubes OS
 %if 0%{?have_uefi}
     -Dplugin_uefi_capsule=true \
     -Dplugin_uefi_pk=true \
-    -Defi_sbat_distro_id="fedora" \
-    -Defi_sbat_distro_summary="The Fedora Project" \
-    -Defi_sbat_distro_pkgname="%{name}" \
-    -Defi_sbat_distro_version="%{version}" \
-    -Defi_sbat_distro_url="https://src.fedoraproject.org/rpms/%{name}" \
     -Dplugin_tpm=true \
+    -Defi_binary=false \
```

Comment:
is that right? Don't we want the subproject like arch? Maybe that's not a huge win just to see if it compiles.

Reply:
Actually maybe even in arch we don't want it to build in.  IE we want all packaged builds to have the EFI binary pulled in separately, and hand builds to pull it in if you don't already have it.

Which brings another question - do we maybe want to distribute a `fwupd-efi.pc` to indicate version?  That would mean we could avoid using the `fs` module all together!

Reply:
So my worry here is that it's really a _runtime_ dep, not a _buildtime_ dep, right?

Reply:
In our previous self's infinite wisdom we actually have a runtime check already:
https://github.com/fwupd/fwupd/blob/master/plugins/uefi-capsule/fu-uefi-device.c#L447

Reply:
We were clever, weren't we :)